locales/en-US/app.l20n

```diff
@@ -104,8 +103,10 @@
 <contribute "Contribute">
 <nowActive "Now Active">
 <userCount title:"Active Users of this Experiment">
-
 <tourOnboardingTitle "{{title}} enabled!">
 <tourStartButton "Take the Tour">
 <tourCancelButton "Skip">
 <tourDoneButton "Done">
+<userCountContainerIntro "There are">
+<userCountContainerOutro "people trying {{title}} right now!">
```

Comment:
Yeah, this should be put together as one string. Both the title and the count can come from the model. The problem is that other languages will need to shuffle this into an entirely different order than English